Example/Doc/InstallationGuide.md

```diff
@@ -0,0 +1,227 @@
+# 🛠️ Installation Guide – NGSI Kafka Stream Processor
```

Comment:
Instead of under `/Example` it would be nice renamed to something like "kafka-ngsi-stream" ?